samza-core/src/main/java/org/apache/samza/clustermanager/ClusterBasedJobCoordinator.java

```diff
@@ -147,8 +161,17 @@ public ClusterBasedJobCoordinator(Config coordinatorSystemConfig) {
 
     metrics = new MetricsRegistryMap();
 
-    //build a JobModelReader and perform partition assignments.
-    jobModelManager = buildJobModelManager(coordinatorSystemConfig, metrics);
+    // Initialize CoordinatorStreamSystem to share between JobModelManager and ChangelogPartitionManager
+    CoordinatorStreamSystemConsumer coordinatorSystemConsumer = new CoordinatorStreamSystemConsumer(coordinatorSystemConfig, metrics);
```

Comment:
Where is this used?

Reply:
Nowhere 😄 . Forgot to delete them. Good catch!